doc/sphinx/language/core/conversion.rst

```diff
@@ -4,11 +4,83 @@ Conversion rules
 ----------------
 
 Coq has conversion rules that can be used to determine if two
-terms are equal by definition, or :term:`convertible`.
+terms are equal by definition in |CiC|, or :term:`convertible`.
 Conversion rules consist of reduction rules and expansion rules.
+
+Conversion rules can be used
+to determine if two terms are convertible by
+converting both terms to a normal form, then verifying they are syntactically
+equal (ignoring differences in the names of bound variables by
+:term:`alpha-convertibility <alpha-convertible>`).
+
 See :ref:`applyingconversionrules`,
 which describes tactics that apply these conversion rules.
 
+Conversion rules can be reductions or expansions.
+Reductions convert terms to something that is incrementally closer to its
+normal form.  For example, :term:`zeta-reduction` removes
+:n:`let @ident := @term__1 in @term__2` constructs from a term by replacing
+:n:`@ident` with :n:`@term__1` wherever :n:`@ident` appears in :n:`@term__2`.
+The resulting term may be longer or shorter than the original.
+
+.. coqtop:: all
+
+   Eval cbv zeta in let i := 1 in i + i.
+
+Expansions are reductions applied in the opposite direction,
+for example expanding `2 + 2` to `let i := 2 in i + i`.  While applying
+reductions gives a unique result, the associated
+expansion may not be unique.  For example, `2 + 2` could also be
+expanded to `let i := 2 in i + 2`.  Reductions that have a unique inverse
+expansion are also referred to as *contractions*.
+
+The normal form is defined as the result of applying a particular
```

Comment:
Same question for normal form.

Reply:
Indeed. Head normal form or weak-head normal forms are just specialized versions of a normal form where we don't reduce everywhere.

Reply:
So I won't add it as a defined term.  Right?

Reply:
So no change needed here, right?

Reply:
On the contrary, I was still suggesting inserting `:term:` here. But if you don't like this proposal, I'm not going to fight :wink: